base_vat_country/tests/test_company_vat_without_country.py

```diff
@@ -0,0 +1,30 @@
+from odoo.tests.common import TransactionCase
+
+
+class CompanyVatWithoutCountry(TransactionCase):
+
+    def setUp(self):
+        super(CompanyVatWithoutCountry, self).setUp()
+        self.company_obj = self.env['res.company']
+        self.mxn = self.env.ref('base.mx')
```

Comment:
It's better to keep using mx instead of mxn. MX is the code for the country, MXN is the code for the currency.

```suggestion
        self.mx = self.env.ref('base.mx')
```

Reply:
@carlosrve 

I change it, because I  get a warning of type pylint 

```bash
[invalid-name] Attribute name "mx" doesn't conform to '[a-z_][a-z0-9_]{2,45}$' pattern [python/pylint] 
```

Reply:
Change applied